### DISCUSSION TITLE
pcb gerber noob

### DISCUSSION THREAD

**Discussion by jonnypoko** [BlueSCSI/BlueSCSI-v2#132] [Build it Yourself]
hi all, brand new tinkerer, tried to upload the 12 gerber files to PCBWay (for DB25 and desktop) to build my own and it failed the auto audit as it 'didnt have a drill file'? obviously the two .drl files are there - what am I doing wrong? just point me towards an idiots guide - ta. There is a PCBWay project page for each already but that seems to be earlier board revisions, is that right?

**Comment by erichelgeson** [comment]
Hi @jonnypoko - You do not upload the gerber files - you normally upload the `...Gerber.zip` containing all the gerbers. I've never used PCBWay - recommend JLCPCB as they are cheaper. 

If you are manufacturing them with the pick and place components look under the Production Files folder for the bill of materials (BOM) and placement files.

**Reply by jonnypoko** [reply]
Hi @erichelgeson Thanks for the reply, i didnt find the zip file until later - I think I was looking for 'gerber' in the filename. The zip file still doesnt seem to have the drill mark file either? - I havent tried to upload them to JLCPCB so I will let you know how I get on. I think I will try to do the soldering myself I have some SMD experience just not much PCB fab chops (yet).
